### ISSUE TITLE
.dat file missing on Linux Flatpack. . 

### ISSUE THREAD

**Issue by FireNWater** [k4zmu2a/SpaceCadetPinball#223]
![Screenshot from 2024-10-28 16-41-22](https://github.com/user-attachments/assets/9f209b47-6d19-48a8-aab3-a4abff8457e7)
.
Flatpack was working until about a week ago, now get this dialog box.  Don't know where to get a .dat file for Linux.
.
Thanks!!

**Comment by Ponali** [comment]
i'm not the project maintainer, but i've found in another issue that you need to find the .dat file yourself.
the .dat file doesn't need any change for the operating system, it stays the same despite the installation.
try to find a copy of Space Cadet Pinball, and put PINBALL.DAT or CADET.DAT into one of the folders listed by the error, and once you're done, launch the Space Cadet flatpak app.
you can also simply extract every asset if you're precautious.

**Comment by FireNWater** [comment]
Yes, I saw that comment also but I've been playing it for a month and now the .dat file is gone. 

I'll go hunt one down.

**Comment by MezzoSplash** [comment]
Same problem here
![image](https://github.com/user-attachments/assets/33a88633-7420-4d5f-8a2c-9365088ebbf4)

**Comment by ScoobyDouche** [comment]
same issue here why are the .dat suddenly missing anyone figure how to fix?

**Comment by FireNWater** [comment]
Download the file from here:  https://archive.org/details/pinballxp

Then extract pinball.dat and all of the sound files into one of the folders.

**Comment by NotHyper-474** [comment]
There seems to have been an update to the Flatpak app and now the error is gone for me. Does it also fix it for someone else?

**Comment by MezzoSplash** [comment]
Yes, I can confirm that the update fixed the Flatpak app for me.

**Comment by Triloworld** [comment]
Issue in MacOS build still. .dat file and other one missing

**Comment by kowalski7cc** [comment]
Hello, I didn't see this issue but it  was related to [Archive.org issues](https://github.com/flathub/com.github.k4zmu2a.spacecadetpinball/issues/31). This is now solev and thus the issue can be closed as solved.

**Comment by t14redhu-star** [comment]
the issue has since reappeared for the flatpak again. 
```
flatpak run com.github.k4zmu2a.spacecadetpinball
Game version: 2.1.1 DEV
Command line:
Compiled with: SDL 2.30.12; SDL_mixer 2.8.0; ImGui 1.89.2 WIP HW
Using SDL renderer: opengl
Loading game from: /app/bin/PINBALL.DAT
```
